libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -4548,3 +4548,34 @@ string YulUtilFunctions::externalCodeFunction()
 		.render();
 	});
 }
+
+std::string YulUtilFunctions::externalFunctionPointersEqualFunction(
+	std::string addressCleanUpFunction,
+	std::string selectorCleanUpFunction
```

Comment:
Don't pass these in as strings - just use ``cleanupFunction(...)`` with the appropriate type below directly - the type will always be the same, no reason to extract it specifically from ``.part(...)``.